src/Microsoft.CodeAnalysis.Analyzers/Core/CodeAnalysisDiagnosticsResources.resx

```diff
@@ -398,4 +398,13 @@
   <data name="DoNotUseReservedDiagnosticIdTitle" xml:space="preserve">
     <value>Do not use reserved diagnostic IDs.</value>
   </data>
+  <data name="DoNotUseCompilationGetSemanticModelDescription" xml:space="preserve">
+    <value>This call leads the compiler to rebind the entire file, and doing it multiple times (say with SyntaxNode callbacks) can be a performance hog.</value>
```

Comment:
I think we need a more refined comment here. @333fred @sharwell do you have a suggestion on how to word this correctly?